plugins/destination/s3/client/spec.go

```diff
@@ -29,6 +29,10 @@ func (s *Spec) SetDefaults() {
 	if s.Delimiter == 0 {
 		s.Delimiter = ','
 	}
+	if s.IncludeHeaders == nil {
+		headers := true
+		s.IncludeHeaders = &headers
```

Comment:
ptr.To(true) (https://pkg.go.dev/github.com/candiduslynx/ptr#To) 😄

Reply:
Yeah that could work, but overall it is the same number of LOC...